include/seqan3/range/concept.hpp

```diff
@@ -48,11 +48,14 @@ SEQAN3_CONCEPT const_iterable_range =
     (std::ranges::random_access_range<std::remove_const_t<type>> == std::ranges::random_access_range<type const>);
 //!\endcond
 
+inline namespace deprecated
```

Comment:
Why do we use a an deprecated inline namespace instead of the deprecation macro?

Reply:
you can't deprecate a concept, they don't accept an attribute

Reply:
meh

Reply:
I removed the concept completely and added a CHANGELOG entry.